### PR TITLE
Tweaks sethome

### PR DESCRIPTION
**Pull request by Rui914**
･Write only at the time of shutdown
･Tab indent
･Minor tweaks

### CODE REVIEW

**Comment by rubenwardy** [comment]
You should also write every ten seconds if there has been a change, as the server may have crashed.

File: mods/sethome/init.lua [hunk 1]
```diff
@@ -1,65 +1,63 @@
...
-            io.close(output)
-            changed = false
-        end
-    end,
+	description = "Set your home point",
+	privs = {home=true},
+	func = function(name)
+		local player = minetest.get_player_by_name(name)
+		if not player then
+			-- just a check to prevent the server crashing
```
Comment (rubenwardy) [inline mods/sethome/init.lua]: no need for a comment here, it's obvious.

File: mods/sethome/init.lua [hunk 2]
```diff
@@ -1,65 +1,63 @@
...
-        else
-            minetest.chat_send_player(name, "Set a home using /sethome")
-        end
-    end,
+	description = "Teleport you to your home point",
+	privs = {home=true},
+	func = function(name)
+		local player = minetest.get_player_by_name(name)
+		if not player then
+			-- just a check to prevent the server crashing
```
Comment (rubenwardy) [inline mods/sethome/init.lua]: same here

File: mods/sethome/init.lua [hunk 3]
```diff
@@ -1,65 +1,63 @@
-local homes_file = minetest.get_worldpath() .. "/homes"
+local homes_file = minetest.get_worldpath().."/homes"
```
Comment (PilzAdam) [inline mods/sethome/init.lua]: Keep the spaces here; there should always be spaces around binary operators.

**Comment by PilzAdam** [comment]
As @rubenwardy pointed out, on_server_shutdown is not a reliable way to store such data.

**Comment by Rui914** [comment]
#669

**Pull request closed without merging**